Documentation/ceph-upgrade.md

```diff
@@ -60,32 +60,26 @@ cd $YOUR_ROOK_REPO/cluster/examples/kubernetes/ceph/
 
 Unless your Rook cluster was created with customized namespaces, namespaces for Rook clusters
 created before v0.8 are likely to be `rook-system` and `rook`, and for Rook-Ceph clusters created
-with v0.8, `rook-ceph-system` and `rook-ceph`. With this guide, we do our best not to assume the
-namespaces in your cluster. To make things as easy as possible, modify and use the below snippet to
-configure your environment. We will use these environment variables throughout this document.
+with v0.8 or after, `rook-ceph-system` and `rook-ceph`. With this guide, we do our best not to
+assume the namespaces in your cluster. To make things as easy as possible, modify and use the below
+snippet to configure your environment. We will use these environment variables throughout this
+document.
 ```sh
 # Parameterize the environment
 export ROOK_SYSTEM_NAMESPACE="rook-ceph-system"
 export ROOK_NAMESPACE="rook-ceph"
 ```
 
-We should start up the new toolbox pod before moving on, and this can be our first test of the
-namespace environment variables.
-```sh
-kubectl -n $ROOK_NAMESPACE delete pod rook-ceph-tools
-kubectl -n $ROOK_NAMESPACE create -f toolbox.yaml
-```
-
 In order to successfully upgrade a Rook cluster, the following prerequisites must be met:
 * The cluster should be in a healthy state with full functionality.
   Review the [health verification section](#health-verification) in order to verify your cluster is
   in a good starting state.
 * All pods consuming Rook storage should be created, running, and in a steady state. No Rook
   persistent volumes should be in the act of being created or deleted.
 
-The minimal sample v0.8 Cluster spec that will be used in this guide can be found below (note that
+The minimal sample v0.9 Cluster spec that will be used in this guide can be found below (note that
 the specific configuration may not be applicable to all environments):
-```
+```yaml
```

Comment:
Could we remove this sample yaml? Since our goal is to upgrade all clusters, I'm not sure what we're showing here.